app/assets/stylesheets/custom.scss

```diff
@@ -247,10 +249,101 @@ body.has-map  {
   }
 }
 
+/*=================================================
```

Comment:
Sería mejor poner los estilos de login/signup en un archivo diferente
https://stackoverflow.com/questions/41400303/how-to-load-style-scss-file-in-application-css-file-ruby-on-rails

Reply:
Okis, me lo miro. Estuve pensando en hacerlo pero por alguna razón el pipeline de carga no me funcionó con los archivos js (las carpetas lib y vendor no eran tenidas en cuenta)  así que decidí de momento tenerlo todo en un solo archivo hasta que quedasemos y poder comentarlo in situ.

En cualquier caso, entiendo que require_self en este caso (archivos de estilos) hace referencia al propio application.css que automaticamente carga custom.scss, planets_dinities.scss y profiles.scss ?

Reply:
Prueba con @import url("file.scss");